YahooStockData/TradeStation/ts_requirements.py

```diff
@@ -0,0 +1,88 @@
+
+
+from os import walk
+import os
+
+file_path = '~/Downloads/'
+easy = 'easy.txt'
+threshold = 'threshold.txt'
+margin = 'margin.txt'
+
+
+def prepare_files():
+    f = []
+
+    global file_path, threshold, margin, easy
+    for (dirpath, dirnames, filenames) in walk(file_path):
+        f.extend(filenames)
+        break
+
+    for file in f:
+        if file.startswith('TSEasy'):
+            os.rename(file_path + file, file_path + easy)
+            print('easy there')
+
+        if file.startswith('TSSpecial'):
+            os.rename(file_path + file, file_path + margin)
+            print('margin there')
```

Comment:
There should be an 'else' statement after these if's, in case a file begins with none of these.
Omission (O)
Rank 2- Fix as part of planned debugging sessions